### PR TITLE
Runner class improvements.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

### CODE REVIEW

File: src/Runner/ConsoleApplicationRunner.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        );
+
+        $container = new Container(
+            $config->get('console'),
+            $config->get('providers-console')
+        );
+
+        $container = $container->get(ContainerInterface::class);
+
+        // Register Bootstrap Service Provider
```
Comment (yiiliveext) [inline src/Runner/ConsoleApplicationRunner.php:47]: What is Bootstrap Service Provider?

File: src/Runner/ConsoleApplicationRunner.php [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        $this->registerBootstrap($container, $config->get('bootstrap-console'));
```
Comment (yiiliveext) [inline src/Runner/ConsoleApplicationRunner.php:48]:
```suggestion
        $this->runBootstrap($container, $config->get('bootstrap-console'));
```

File: src/Runner/ConsoleApplicationRunner.php (hunk 1 above)
Comment (terabytesoftw) [reply src/Runner/ConsoleApplicationRunner.php:47]: done.

File: src/Runner/ConsoleApplicationRunner.php (hunk 2 above)
Comment (terabytesoftw) [reply src/Runner/ConsoleApplicationRunner.php]: done.

**Review by samdark — APPROVED** [review]

**Pull request merged**